AMW_angular/pom.xml

```diff
@@ -57,23 +57,23 @@
                         </goals>
                     </execution>
                     <execution>
-                        <id>npm ci</id>
+                        <id>npm install</id>
```

Comment:
Why? Is there a reason to change it from npm ci to npm install? What is the difference between the two of them?

Reply:
we should use `npm ci` here and not `npm install`

See: https://stackoverflow.com/a/53325242

Reply:
Sorry, changed to pnpm in first position, there is no pnpm ci ....npm ci is correct